src/components/Router/index.js

```diff
@@ -31,15 +30,16 @@ class Router extends Component {
   }
 
   render = (props) => {
-    const { websocketErrorEncountered, options, ...otherProps} = props
+    const { options, ...otherProps} = props
     const defaultStepOptions = ['welcome','document','face','complete']
     const stepOptions = options.steps || defaultStepOptions
     otherProps.nextStep = this.nextStep
+    otherProps.token = options.token
+    otherProps.serverUrl = options.serverUrl
```

Comment:
What do you think of this @rfreitas? I wrote it, but I'm not thrilled by it.
The options object contains all user config, but we use most of it in parent components. I then explicitly pull out the options we need to pass below the Router. I feel we are halfway between passing all options to all components and only passing the required options.